### PR TITLE
allow pipe param for pecl extensions

### PR DESCRIPTION
**Pull request by cbaoxa**
cf https://github.com/voxpupuli/puppet-php/blob/master/lib/puppet/provider/package/pecl.rb#L83

### CODE REVIEW

File: manifests/extension.pp [hunk 1]
```diff
@@ -66,6 +70,7 @@
   $settings          = {},
   $settings_prefix   = false,
   $sapi              = 'ALL',
+  $pipe              = undef,
```
Comment (alexjfisher) [inline manifests/extension.pp]: Does `$pipe` then not also have to be passed as an argument to `ensure_packages`?

Comment (cbaoxa) [inline manifests/extension.pp]: Thanks! I ran the wrong test on my side and missed this. PR updated

File: manifests/extension.pp [hunk 2]
```diff
@@ -52,6 +52,10 @@
 #   String parameter, whether to specify ALL sapi or a specific sapi.
 #   Defaults to ALL.
 #
+# [*pipe*]
+#   String parameter that may be used by the *provider* to input answers during
```
Comment (alexjfisher) [inline manifests/extension.pp]: Maybe mention it's for the 'pecl' provider only here?

Comment (cbaoxa) [inline manifests/extension.pp]: Hoping this will be updated correctly in the future, sure :)

**Pull request merged**